datasets/huishoudelijkafval/ticket/v1.0.0.json

```diff
@@ -0,0 +1,249 @@
+                "description": "De datum waarop de werkzaamheden als gereed aangegeven zijn door het onderhoudsbedrijf."            
+             },
+            "datumTechnischeGoedkeuring": {
+                "type": "datetime",
+                "description": "De datum waarop de technische aspecten van de werkzaamheden goed- of afgekeurd zijn." 
+            },
+            "datumFinanciëleGoedkeuring": {
+                "type": "datetime",
+                "description": "De datum waarop de financiële aspecten van de werkzaamheden goed- of afgekeurd zijn." 
+            },
+            "datumGefactureerd": {
+                "type": "datetime",
+                "provenance": "datum_gefactureerd_in_systeem",
+                "description": "De datum waarop de factuur in het systeem de status 'gefactureerd' krijgt."
+            },
+            "datumAfgewezen": {
+                "type": "datetime",
+                "description": "De datum waarop het ticket de status 'afgewezen' krijgt."
+            },
+            "redenAfgewezen": {
+                "type": "string",
+                "provenance": "afgewezen_reden",
+                "description": "De reden waarom het ticket de status 'afgewezen' heeft gekregen."
+              },
+            "prijs": { 
+                "type": "string",
+                "description": "De totale prijs van de gebruikte onderdelen en de uitgevoerde werkzaamheden."
+            },
+            "prijsOfferte": { 
+                "type": "string",
+                "description": "De totale prijs op de offerte voor de te gebruiken onderdelen en de uit te voeren werkzaamheden."
+            },
+            "doorlooptijd": {
+                "type": "string",
+                "description": "De tijdsspanne in werkdagen tussen de 'datumCreatie'en de 'datumAfgerond' of de tijdsspanne in werkdagen tussen de 'datumCreatie' en de laatste statuswijziging in het geval het ticket nog niet afgerond is."
+            },
+            "ticketItemsDefinitief": {
+                "type": "string",
+                "description": "De onderdelen en de werkzaamheden zoals deze op de factuur vermeld worden."
+            },
+            "ticketItemsOfferte": {
+                "type": "string",
+                "description": "De onderdelen en de werkzaamheden zoals deze op de offerte vermeld worden."
+            }, 
+            "gbdBuurt": {
+                "type": "string",
+                "relation": "gebieden:buurten",
+                "uri": "https://www.amsterdam.nl/stelselpedia/gebieden-index/catalogus/buurt",
+                "description": "Unieke identificatie van het object"
+            },
+            "gbdBuurtNaam": {
+                "type": "string",
+                "uri": "https://www.amsterdam.nl/stelselpedia/gebieden-index/catalogus/buurt",
+                "description": "De naam van het object." 
+            },
+            "gbdBuurtVolgnummer": {
+                "type": "string",
```

Comment:
The `volgnummer` in the `gebieden` dataset is always an `integer`.

Reply:
aangepast